.travis.yml

```diff
@@ -5,71 +5,80 @@ services:
 
 language: go
 
-go:
-  - 1.7.1
-
 # Make sure project can also be built on travis for clones of the repo
 go_import_path: github.com/elastic/beats
 
-os:
-  - linux
-  - osx
-
 env:
-  matrix:
-    - TARGETS="check"
-    - TARGETS="-C filebeat testsuite"
-    - TARGETS="TEST_ENVIRONMENT=0 -C filebeat testsuite"
-    - TARGETS="-C heartbeat testsuite"
-    - TARGETS="-C libbeat testsuite"
-    - TARGETS="-C metricbeat testsuite"
-    - TARGETS="TEST_ENVIRONMENT=0 -C metricbeat testsuite"
-    - TARGETS="-C packetbeat testsuite"
-    - TARGETS="-C libbeat crosscompile"
-    - TARGETS="-C metricbeat crosscompile"
-    - TARGETS="-C winlogbeat crosscompile"
-    - TARGETS="-C libbeat/dashboards"
-    - TARGETS="-C generate/metricbeat/metricset test"
-    - TARGETS="-C generate/beat test"
   global:
     # Cross-compile for amd64 only to speed up testing.
     - GOX_FLAGS="-arch amd64"
-    - DOCKER_COMPOSE_VERSION: 1.8.1
+    - DOCKER_COMPOSE_VERSION: 1.9.0
+    - &go_version 1.7.1
 
 matrix:
-  exclude:
-    - os: osx
+  include:
+    # General checks
+    - os: linux
       env: TARGETS="check"
+      go: *go_version
```

Comment:
Unfortunately this mix of global go version and include doesn't work :-( It will not read the global golang version.